packages/node-sdk/example/app.ts

```diff
@@ -0,0 +1,73 @@
+import express from "express";
+import { bucket } from "./bucket";
+import { BucketClient } from "../src";
+import type { AppFlags } from "./flags";
+
+export const app = express();
+
+declare global {
+  namespace Express {
+    interface Locals {
+      bucketUser: BucketClient;
+    }
+  }
+}
+
+app.use(express.json());
+app.use((req, res, next) => {
+  const bucketUser = bucket
+    .withUser(req.headers["x-bucket-user-id"] as string, {
+      attributes: {
+        role: req.headers["x-bucket-is-admin"] ? "admin" : "user",
+      },
+    })
+    .withCompany(req.headers["x-bucket-company-id"] as string, {
+      attributes: {
+        betaUser: !!req.headers["x-bucket-company-beta-user"],
+      },
+    });
+
+  res.locals.bucketUser = bucketUser;
+  next();
+});
+
+const todos = ["Buy milk", "Walk the dog"];
+
+app.get("/", (_req, res) => {
+  res.locals.bucketUser.trackFeatureUsage("Front Page Viewed");
+  res.send("Hello World");
+});
+
+app.get("/todos", async (_req, res) => {
+  const flags = res.locals.bucketUser.getFlags<AppFlags>();
+
+  if (flags["show-todos"]) {
+    res.locals.bucketUser.trackFeatureUsage("Front Page Viewed");
+  }
+
+  res.json({ todos });
+});
+
+app.post("/todos", (req, res) => {
+  const { todo } = req.body;
+  if (typeof todo !== "string") {
+    return res.status(400).json({ error: "Invalid todo" });
+  }
+
+  todos.push(todo);
+
```

Comment:
let's add a "create-todos" flags which gates the creation of flags? There's only one flag atm. and the functionality doesn't make that much sense.

Reply:
and maybe one for "delete-todos" too?